libs/model-assessment/src/lib/ModelAssessmentDashboard/Cohort/CohortList.tsx

```diff
@@ -178,13 +179,15 @@ export class CohortList extends React.Component<
                     </Stack.Item>
                   </Stack>
                 </Stack.Item>
-                <Stack.Item>
-                  <CohortListDeleteButton
-                    disabled={this.isActiveCohort(item.key)}
-                    itemKey={item.key}
-                    onDeleteCohortClick={this.onDeleteCohortClick}
-                  />
-                </Stack.Item>
+                {this.props.enableEditing && (
```

Comment:
for my knowledge: why is this added now?

Reply:
Great question! Cohorts (other than "All data") can generally be edited. Forecasting is a bit of an oddball there in that not all cohorts are really valid as time series. Time series require filters based on "time series id columns." So rather than making users create these by hand (which is prone to errors) we generate all of them ahead of time and pass them into the dashboard. That means they don't need to have editing capabilities as that wouldn't accomplish anything. There is a special case where one might want to merge some of these together, but that's not in scope for the MVP. [I'm planning to pass in some merged time series of interest at a later point if users are interested in that.]

Reply:
Resolving this one, but do reopen if you have questions.